### ISSUE TITLE
That's strange

### ISSUE THREAD

**Issue by parro-it** [ewnd9/electron-remote-require-investigation#1]
what I get is instead:

```
b is required
b is not remote required Error: Cannot find module 'b'(…)(anonymous function)
c is required
c is remote required
```

**Comment by parro-it** [comment]
No, wait never mind. I was using NPM ver 3 and it move module B

**Comment by parro-it** [comment]
Same output as you

**Comment by parro-it** [comment]
Documentation sais: 

> remote.require(module)
> 
> module String
> Returns the object returned by require(module) in the main process.

I think this is a bug on electron side... I'll open an issue, or do you want to do yourself?

**Comment by ewnd9** [comment]
> I'll open an issue, or do you want to do yourself?

Please, do it by yourself.

**Comment by parro-it** [comment]
👍

**Comment by parro-it** [comment]
https://github.com/electron/electron/issues/5090
